webpack/assets/javascripts/react_app/components/componentRegistry.js

```diff
@@ -32,6 +32,7 @@ import TemplateInput from './Template/TemplateInput';
 
 // Pages
 import AuditsPage from '../pages/AuditsPage/AuditsPage';
+import HostWizardPage from '../pages/HostWizardPage';
```

Comment:
there is no need for the page to be in the registry, since the router will mount it and will recognise it.
use `PageLayout`, take a look at my Audits PR